src/SubNav.js

```diff
@@ -46,19 +46,19 @@ const SubNav = styled(SubNavBase)`
   ${sx};
 `
 
-SubNav.Links = Flex
+SubNav.Links = props => <Flex {...props} />
```

Comment:
It's been a while since I brushed up on JS gotchas, but I was under the impression that doing `SubNav.Links = Flex` wouldn't cause a reassignment of `Flex` but I guess I was wrong!